packages/cli-poa/src/driver.js

```diff
@@ -0,0 +1,40 @@
+import fetch from 'node-fetch';
```

Comment:
can we leverage `sdk-utils/request.js` its stripped request lib and we won't have to deal with dependency in future.

Reply:
Tried this out, but received the error -

```
{'build': {'id': '26967368', 'url': 'https://percy.io/test/app-amit-debugs/builds/26967368', 'number': 489}, 'error': 'Protocol "https:" not supported. Expected "http:"', 'success': False}
```